tests/phpunit/tests/Story_Renderer.php

```diff
@@ -76,12 +76,12 @@ public function test_replace_body_start_tag() {
 			echo 'Hello World';
 		};
 
-		add_action(	'web_stories_body_open', $function );
+		add_action( 'web_stories_body_open', $function );
```

Comment:
I have no idea what these changes are doing here. But the pre-commit complained, I let it auto-fix, and here we are.

Reply:
Odd that this wasn‘t caught in my PR 🤔

But the change is fine.